src/site/filters/liquid.js

```diff
@@ -90,7 +90,7 @@ module.exports = function registerFilters() {
   liquid.filters.genericModulo = (i, n) => i % n;
 
   liquid.filters.listValue = data => {
-    const string = data.split('_').join(' ');
+    const string = data && data.length ? data.replace('_', ' ') : data;
```

Comment:
Nice.

Reply:
The naming here feels a bit confusing.  It isn't really clear what `listValue` will do, but the name suggests that `data` is supposed to be an array.  However, the actual code of the function makes it seem like `data` is actually a string since there isn't a `replace()` method for arrays.  Can you change the function name to something like `removeUnderscores`?